index.d.ts

```diff
@@ -0,0 +1,25 @@
+declare module 'golden-colors' {
+  export function getHsvGolden(s: number, v: number): WebColor;
+
+  export function getHsvSimple(s: number, v: number): WebColor;
+
+  export function getNaive(): any;
+
+  export interface WebColor {
+    r: number;
+    g: number;
+    b: number;
+
+    new(r: number, g: number, b: number): WebColor;
```

Comment:
These lines probably should be removed, because the module doesn't allow instantiate `WebColor` outside the module and the `r`, `g`, and `b` properties are not exposed.